sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Utils.scala

```diff
@@ -55,4 +57,11 @@ private[sql] object DataSourceV2Utils extends Logging {
 
     case _ => Map.empty
   }
+
+  // SPARK-15895: Metadata files (e.g. Parquet summary files) and temporary files should not be
```

Comment:
Why do you use `DataSourceV2Utils` instead of `DataSourceUtils`?

Reply:
Moved it, I couldn't see that file before pulling the upstream master.